src/content/developers/docs/networks/index.md

```diff
@@ -57,7 +57,7 @@ The Sepolia network uses a permissioned validator set. It's fairly new, meaning
 - [QuickNode Sepolia Faucet](https://faucet.quicknode.com/drip)
 - [Grabteeth](https://grabteeth.xyz/)
 - [PoW faucet](https://sepolia-faucet.pk910.de/)
-- [Sepolia faucet](https://faucet.sepolia.dev/)
+- [Alternative PoW faucet](https://faucet.sepolia.dev/)
```

Comment:
```suggestion
```
I think we just get rid of this faucet as well. The link wasn't updated here, but I went and checked it out and the faucet had no ETH in it for users either. I don't imagine its the most reliable.